packages/examples/src/geometric-queries/closest-silhouette-point/test-closest-silhouette-point.substance

```diff
@@ -0,0 +1,11 @@
+-- variation: TopsailBoar92821
```

Comment:
Is there a reason you write a variation here different from the one in the corresponding `*.trio.json` file?